test/tests/functional/pbs_reliable_job_startup.py

```diff
@@ -3132,6 +3131,10 @@ def test_t7(self):
                                  'schedselect': self.job1_ischedselect},
                            id=jid, attrop=PTL_AND)
 
+        # tolerate_node_failures=none and launch hook calls release_nodes()
+        emsg = "no nodes released as job does not tolerate node failures"
+        self.momA.log_match("%s: %s" % (jid, emsg), n=30, max_attempts=10)
```

Comment:
I wouldn't set max_attempts.  You might run into a race condition on slow machines.